.ci.yaml

```diff
@@ -2586,6 +2597,31 @@ targets:
         ["devicelab", "hostonly", "mac"]
       task_name: flutter_gallery_macos__start_up
 
+  - name: Mac flutter_packaging_test
+    recipe: packaging_v2/packaging_v2
+    timeout: 60
+    bringup: true
+    enabled_branches:
+      - main
+    properties:
+      task_name: flutter_packaging
+      tags: >
+        ["framework", "hostonly", "shard", "mac"]
+
+
+  - name: Mac_arm64 flutter_packaging_test
+    recipe: packaging_v2/packaging_v2
+    timeout: 60
+    bringup: true
+    enabled_branches:
+      - main
```

Comment:
any reason not to run this on release branch pre-submit?

Reply:
Let me double check the filtering logic, but yeah it can run on any branch.

Reply:
Added a few more checks to the recipe to ensure we only upload artifacts if the build is not experimental and running from prod or flutter buckets(official builds.)

Reply:
Recipes change has landed and this PR has been updated. Would you mind taking another look when you have a chance?